src/components/Tabs/Annotation/AnnotationDisplayList.tsx

```diff
@@ -0,0 +1,136 @@
+import { Collapse } from "antd";
+import React, { ReactElement, useContext, useEffect, useMemo, useRef } from "react";
+import styled from "styled-components";
+
+import { TagIconSVG } from "../../../assets";
+import { Dataset } from "../../../colorizer";
+import { FlexColumn, FlexColumnAlignCenter, FlexRowAlignCenter } from "../../../styles/utils";
+
+import { AppThemeContext } from "../../AppStyle";
+import AnnotationDisplayTable, { TableDataType } from "./AnnotationDisplayTable";
+
+type AnnotationDisplayListProps = {
+  dataset: Dataset | null;
+  ids: number[];
+  onClickObjectRow: (record: TableDataType) => void;
+  onClickDeleteObject: (record: TableDataType) => void;
+};
+
+const StyledCollapse = styled(Collapse)`
+  &&&& .ant-collapse-header {
```

Comment:
Out of curiosity, when do you specify styling like this, rather than in a separate .css file?

Reply:
I've been migrating away from separate .css files and I've been instead doing either inline styling on the component (the `styles={{...}}`) or styled components like you see here. There's a lower chance for classes etc. to become mismatched between your JSX and your CSS this way!